### PR TITLE
Function to pull general vaccine data and adult pop

### PR DESCRIPTION
**Pull request by hjohns12**
Fetch latest state vaccine numbers and 18+ population estimate from 2020 census

Didn't know how to add to this existing PR, my bad!

### CODE REVIEW

**Comment by hjohns12** [comment]
PS, I see the [build check is failing](https://github.com/uclalawcovid19behindbars/behindbarstools/pull/127/checks?check_run_id=3231219898#step:10:788) on `calc_aggregate_counts`. Unclear to me why this is happening since that function didn't change...?

File: R/get_genstate_vax.R [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#'
+#' @description Pulls the latest data from the CDC on COVID vaccinations initiated
+#' for adults in a state and then merges alongside a population number from the 18+ 2020
+#' Census population estimates for that state to use for rate calculation.
+#'
+#' @return a data frame with the following columns: State, Date,
+#' General.Initiated, General.Adult.Population
+#'
+#' @examples
+#' \dontrun{
```
Comment (erika-tyagi) [inline R/get_genstate_vax.R:11]: I wouldn't be opposed to running this example as part of the build! Never know when the CDC or Census might change links, etc. on us, and it's fast to run!

File: R/get_genstate_vax.R [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+               General.Adult.Population = POPEST18PLUS2020) %>%
+        mutate(State = ifelse(State == "United States", "National", State))
+
+    raw_cdc_vax <- str_c(
+        "https://covid.cdc.gov/covid-data-tracker/COVIDData/",
+        "getAjaxData?id=vaccination_data") %>%
+        jsonlite::read_json(simplifyVector = TRUE)
+
+    gen_vax_df <- as_tibble(raw_cdc_vax$vaccination_data) %>%
+        mutate(General.Initiated = Administered_Dose1_Recip_18Plus)  %>%
```
Comment (erika-tyagi) [inline R/get_genstate_vax.R:36]: Thoughts on also adding `General.Completed` here? I think it corresponds to `Series_Complete_18Plus` in the CDC data (cross-checked a few numbers and this seemed right, but always good to verify)!

File: R/get_genstate_vax.R [hunk 3]
```diff
@@ -0,0 +1,46 @@
+#' Get general population state COVID vaccine data from CDC and Census 2020 18+ pop estimates
+#'
+#' @description Pulls the latest data from the CDC on COVID vaccinations initiated
+#' for adults in a state and then merges alongside a population number from the 18+ 2020
+#' Census population estimates for that state to use for rate calculation.
```
Comment (erika-tyagi) [inline R/get_genstate_vax.R:5]: Realllyyy little thing, but thoughts on adding something to the docs saying this is for adults 18+? Feels like something I will personally forget at some point 😭

**Review by erika-tyagi — COMMENTED** [review]
Lgtm, just added a few minor comments. And I think the previous build failed on the `read_scrape_bug`, but re-running the job seems to have worked!

File: R/get_genstate_vax.R (hunk 2 above)
Comment (hjohns12) [reply R/get_genstate_vax.R:36]: Good idea, thanks! Adding this in now

**Review by erika-tyagi — APPROVED** [review]

**Pull request merged**